projects/pdarm.md

```diff
@@ -0,0 +1,84 @@
+
+## Project Name 
+  
+Parametric Digital Asset Risk Management
+
+## Category
+
+DeFi risk management, dApp, IPFS
+
+## Project Description
+
+A dApp which provides parametric stablecoin lending protection on Aave (other DeFi platforms TBD), using IPFS+Chainlink+TUSD+Aave.
+
+For additional information about our project please see our [Devpost project page](https://devpost.com/software/parametric-digital-asset-risk-management) where we discuss the following:
+
+-   About the project    
+-   Inspiration
+-   The Problem
+-   What it does
+-   How we built it
+-   Challenges we ran into
+-   Accomplishments that we're proud of
+-   What we learned
+-   What's next for Parametric Digital Asset Risk Management
+-   How we integrated with sponsors
+
+## Use of IPFS, Filecoin and Libp2p
+
+We used IPFS to deploy and host our project’s application, website, store and display images on the website, and to display smart contract analytical data on user dashboards.  
+
+## Project Status
+
+Under development  
+
+## Previews
+
+-   [5 Minute Pitch and Demo video](https://www.youtube.com/watch?v=nHXhbZFmX6M)
+-   [Please check out our 7 minute slowed down and extended demo video!](https://youtu.be/tqz92QOn9kw)
+-   [IPFS Live Demo](https://ipfs.io/ipfs/QmdY4Fq33R4tnnMDR4spSPeT7ony4ZSd7Gs7Rp9rAEwDQ5/)  
+
+## Target Audience
+
+The current audience is Non-DeFi users who seek to interact with DeFi lending protocols in a manner that has circuit breakers and guard rails, and DeFi natives who seek risk-adjusted yields.
+
+## Rough estimated user base (if applicable)
+
+[1, inf) users  
+
+## Github repo
+
+https://github.com/chainlink-hackathon2021-insurance/DeFi-Lending-Insurance
+
+## Website
+
+IPFS
+https://ipfs.io/ipfs/QmdY4Fq33R4tnnMDR4spSPeT7ony4ZSd7Gs7Rp9rAEwDQ5/
+  
+
+## Docs
+
+Devpost Submission: https://devpost.com/software/parametric-digital-asset-risk-management
+
+## Team Info
+
+We are a team of four blockchain enthusiasts and builders seeking to expand on an ambitious concept using IPFS as one of our core pillars.
+
+### Team Size
+
+4
+
+### Team members
+
+-   Seth Andrus: System Architecture
+-   Luis Ramírez: Smart Contract & Frontend Development
+-   Mike Robinson: Product Manager & Market Researcher
+-   Lukas Sexton: UI/UX design and research
+
+## How the community can engage
```

Comment:
can you create a discussion thread in the ipfs/community discussion board and add link in this section? instructions here: https://github.com/ipfs/community#ecosystem-projects

Reply:
Sure. I have created the discussion thread and added the link here.